testsuite/openshift/client.py

```diff
@@ -213,6 +213,15 @@ def select_operator(apiobject):
 
         return self.select_resource("pods", narrow_function=select_operator)
 
+    def get_apicast_operator(self):
+        """
+        Gets the selector for the apicast operator
+
+        :return: the operator pod
+        """
+
+        return self.select_resource("subscriptions")
```

Comment:
What if there will be multiple namespace operators? Will it still select only the apicast-operator?

Reply:
That's why it's Draft, as this is for our current setup, where apicast-operator is only operator in namespace.

Reply:
Added selector by the name